agent/exploits/cve_2019_12989.py

```diff
@@ -77,7 +76,7 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
                 return [vulnerability]
             else:
                 return []
-        except json.JSONDecodeError:
+        except requests.exceptions.JSONDecodeError:
```

Comment:
Why the change?

Reply:
the unit test was failing.

Reply:
That's a red flag, how do I know this is the actual exception raised and not some wrong artifact of the unit test?

Reply:
It's what was shown to me when I ran the unit tests locally, I'll leave it as It was. If it raises again, then we solve it.